content/file-attachment.ts

```diff
@@ -9,7 +9,28 @@ import { DEFAULT_LOCALE } from "../libs/constants";
 import { ROOTS } from "../libs/env";
 import { memoize, slugToFolder } from "./utils";
 
-function isImage(filePath) {
+function isFileAttachment(filePath: string) {
+  return (
+    isAudio(filePath) ||
+    isFont(filePath) ||
+    isVideo(filePath) ||
+    isImage(filePath)
+  );
+}
+
+function isAudio(filePath: string) {
+  return /\.(mp3|ogg)$/i.test(filePath);
+}
+
+function isFont(filePath: string) {
+  return /\.(ttf)$/i.test(filePath);
+}
```

Comment:
We might want to check if the file is actually a font via [is-font](https://www.npmjs.com/package/is-font).

That package supports `eot`, `otf`, `ttf`, `woff` and `woff2`.

Reply:
Or we just use [file-type](https://www.npmjs.com/package/file-type#supported-file-types).

Reply:
I think that's probably a good idea. I imagine we might want to have at least `woff` and `woff2` support.